ELECTRON.md

```diff
@@ -1,7 +1,5 @@
 ## Install
 
-Must use node v8.9.0
```

Comment:
We should change here to use node v14. 

We've found some problems with node v12 and node v16, so it's important to make it clear here.

Or just remove the `Install` section because it's empty now

Reply:
Ok. Yeah! I believe to make clear is the best option. I will change.